Backpack/src/main/java/net/skyscanner/backpack/chip/BpkChip.kt

```diff
@@ -18,20 +18,42 @@ open class BpkChip @JvmOverloads constructor(
   defStyleAttr: Int = 0
 ) : BpkText(wrapContextWithDefaults(context), attrs, defStyleAttr) {
 
-  init {
-    initialize(attrs, defStyleAttr)
-  }
-
   var disabled: Boolean = false
     set(value) {
       field = value
       this.isEnabled = !disabled
     }
 
+  var chipBackgroundColor: Int = -1
+    set(value) {
+      field = value
+      setupBackground()
+    }
+
+  var chipSelectedBackgroundColor: Int = -1
+    set(value) {
+      field = value
+      setupBackground()
+    }
+
+  init {
+    initialize(attrs, defStyleAttr)
+  }
+
   private fun initialize(attrs: AttributeSet?, defStyleAttr: Int) {
-    val attr = context.theme.obtainStyledAttributes(attrs, R.styleable.BpkChip, defStyleAttr, 0)
+    val t = TypedValue()
+    var wrapped: Context = context
+
+    if (context.theme.resolveAttribute(R.attr.bpkChipStyle, t, true)) {
+      // If we have global styles (theming) for the chip we wrap again so this overrides the default
+      wrapped = ContextThemeWrapper(context, t.resourceId)
+    }
+
+    val attr = wrapped.theme.obtainStyledAttributes(attrs, R.styleable.BpkChip, defStyleAttr, 0)
     disabled = attr.getBoolean(R.styleable.BpkChip_disabled, false)
     isSelected = attr.getBoolean(R.styleable.BpkChip_selected, false)
+    chipBackgroundColor = attr.getColor(R.styleable.BpkChip_chipBackgroundColor, ContextCompat.getColor(context, R.color.bpkGray50))
```

Comment:
fallback to the property itself, e.g. chipBackgroundColor 
 = attr.getColor(R.styleable.BpkChip_chipBackgroundColor, chipBackgroundColor)